components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -78,4 +81,72 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testColorThemer() {
+    // Given
+    let titleTest = "Title"
+
+    // When
+    actionSheet.title = titleTest
+
+    // Then
+    if let title = getLabel(with: titleTest) {
+      XCTAssertEqual(title.textColor, .black)
```

Comment:
This doesn't seem related to the "when" condition. Why would changing the title change the color of the text?

Reply:
Make sure the default behavior is as intended. Split into a separate test.